lib/generators/doorkeeper/templates/add_previous_refresh_token_to_access_tokens.rb

```diff
@@ -0,0 +1,5 @@
+class AddPreviousRefreshTokenToAccessTokens < ActiveRecord::Migration
+  def change
+    add_column :oauth_access_tokens, :previous_refresh_token, :string
+  end
+end
```

Comment:
We need to define a way for existing applications to upgrade this. See the rake task for adding scopes to applications for reference.